### DISCUSSION TITLE
Documentation Guide: Directus and Community Resources

### DISCUSSION THREAD

**Discussion by philleepflorence** [directus/directus#4329] [Feature Requests]
## What

A summary of all the Directus and Community Resources.

### What are the Directus Resources

Resources and platforms that support _output_ to the community.

* Marketing Website
    * [https://directus.io](https://directus.io)
    * General Information, Features, and more.
* Blog
    * [https://blog.directus.io](https://blog.directus.io)
    * Articles.
* Documentation
    * [https://docs.directus.io](https://docs.directus.io)
    * Concepts, Guides, and References.
* Knowledge Base
    * [https://blog.directus.io](https://blog.directus.io)
    * Tutorials, Case Studies, Use Cases.
* Market Place
    * [https://directus.market](https://directus.market)
    * Extensions.

### What are the Community Resources

Resources and platforms to support _input_ and _output_ to the community.

* GitHub Discussions
    * [https://github.com/directus/directus/discussions](https://github.com/directus/directus/discussions)
    * Primary community support forum and Q/A hub - API.
* GitHub Issues
    * [https://github.com/directus/directus/issues](https://github.com/directus/directus/issues)
    * Developer support.
* Discord
    * [https://directus.chat](https://directus.chat)
    * Community Chat
* Stackoverflow - _Not Recommended_
    * [https://stackoverflow.com/questions/tagged/directus](https://stackoverflow.com/questions/tagged/directus)
    * Use the Stackoverflow API to control or manage questions related to Directus.
    * [Subscribe to StackOverflow tags like Directus.](https://meta.stackoverflow.com/questions/254318/how-to-subscribe-to-tags#:~:text=Just%20go%20to%20the%20Filtered,you%20filter%20for%20as%20well)

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
